pkg/reconciler/volumeclaim/pvchandler_test.go

```diff
@@ -243,35 +242,3 @@ func TestCreateExistPersistentVolumeClaims(t *testing.T) {
 		t.Fatalf("unexpected PVC name on created PVC; expected: %s got: %s", expectedPVCName, pvcList.Items[0].Name)
 	}
 }
-
-func TestPurgeFinalizerAndDeletePVCForWorkspace(t *testing.T) {
```

Comment:
why was this test deleted? it should still be valid no?

Reply:
It seems like the simple client set cannot mock the pvc with finalizer scenario. Specifically, the `pvcs` are already successfully deleted in the first API call, so the second call to remove finalizer will fail since the `pvc` is already deleted.

I could potentially use some expected errors in the test case to compare, but I feel it is quite confusing. And the functionality is already covered by integration test.

Reply:
got it, one way to do this could be to use something like `PrependReactor` for the delete call and have it set the `deletionTimestamp` instead of deleting the object

https://github.com/tektoncd/pipeline/blob/f5578a8b5191409fa085009be10277c530200a61/pkg/reconciler/pipelinerun/affinity_assistant_test.go#L435-L438

Reply:
Thanks for the advice, I have added the test back with your suggestion. PTAL!